metrics/micrometer/src/main/java/com/datastax/oss/driver/internal/metrics/micrometer/MicrometerSessionMetricUpdater.java

```diff
@@ -30,22 +30,28 @@
 import io.micrometer.core.instrument.Timer;
 import java.time.Duration;
 import java.util.Set;
+import java.util.function.Supplier;
 import net.jcip.annotations.ThreadSafe;
 
 @ThreadSafe
 public class MicrometerSessionMetricUpdater extends MicrometerMetricUpdater<SessionMetric>
     implements SessionMetricUpdater {
+  private final Supplier<Number> connectedNodesSupplier = this::connectedNodes;
```

Comment:
Same here.

Reply:
Same as the similar question asked above.